docs/framework/winforms/controls/errorprovider-component-overview-windows-forms.md

```diff
@@ -10,7 +10,7 @@ helpviewer_keywords:
 ms.assetid: ced189f2-b5c8-46a7-a6f1-37f5af95dc99
 ---
 # ErrorProvider Component Overview (Windows Forms)
-The Windows Forms [ErrorProvider](errorprovider-component-windows-forms.md) component is used to validate user input on a form or control. It is typically used in conjunction with validating user input on a form, or displaying errors within a dataset. An error provider is a better alternative than displaying an error message in a message box, because once a message box is dismissed, the error message is no longer visible. The <xref:System.Windows.Forms.ErrorProvider> component displays an error icon (![ErrorProvider icon](./media/vberrorprovidericon.gif "vbErrorProviderIcon")) next to the relevant control, such as a text box; when the user positions the mouse pointer over the error icon, a ToolTip appears, showing the error message string.  
+The Windows Forms [ErrorProvider](../../../../docs/framework/winforms/controls/errorprovider-component-windows-forms.md) component is used to validate user input on a form or control. It is typically used in conjunction with validating user input on a form, or displaying errors within a dataset. An error provider is a better alternative than displaying an error message in a message box, because once a message box is dismissed, the error message is no longer visible. The <xref:System.Windows.Forms.ErrorProvider> component displays an error icon (![Icon displayed by the ErrorProvider component.](./media/errorprovider-component-overview-windows-forms/vb-error-provider-icon.gif)) next to the relevant control, such as a text box; when the user positions the mouse pointer over the error icon, a ToolTip appears, showing the error message string.  
```

Comment:
```suggestion
The Windows Forms [ErrorProvider](errorprovider-component-windows-forms.md) component is used to validate user input on a form or control. It is typically used in conjunction with validating user input on a form, or displaying errors within a dataset. An error provider is a better alternative than displaying an error message in a message box, because once a message box is dismissed, the error message is no longer visible. The <xref:System.Windows.Forms.ErrorProvider> component displays an error icon (![A white exclamation point inside a red circle.](./media/errorprovider-component-overview-windows-forms/vb-error-provider-icon.gif)) next to the relevant control, such as a text box; when the user positions the mouse pointer over the error icon, a ToolTip appears, showing the error message string.  
```